components/layout/sider.component.ts

```diff
@@ -143,21 +140,6 @@ export class NzSiderComponent implements OnInit, AfterViewInit, OnDestroy, OnCha
     this.updateMenuInlineCollapsed();
   }
 
-  ngAfterViewInit(): void {
-    if (this.platform.isBrowser) {
-      merge(
-        this.nzDomEventService.registerResizeListener().pipe(finalize(() => this.nzDomEventService.unregisterResizeListener())),
-        of(true).pipe(delay(0))
-      )
-        .pipe(takeUntil(this.destroy$))
-        .subscribe(() => {
-          this.ngZone.run(() => {
```

Comment:
should not remove ngZone.run since the resizeSerivce is runOutside

Reply:
Resize emits events in Angular zone:

```ts
    this.ngZone.run(() => {
      this.resizeSource$.next();
    });
```